scripts/export/export_to_trt_llm.py

```diff
@@ -12,148 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import argparse
 import logging
 import sys
-from typing import Optional
 
 from nemo.export.tensorrt_llm import TensorRTLLM
+from nemo.export.tensorrt_llm_parsing_utils import add_export_kwargs, add_multi_block_mode_flag, create_parser
```

Comment:
Let's maybe put these utils into `nemo.export.utils` and argparse.py script there

Reply:
(just to use shorter names and current utils to make it look nicer)